app/pages/routeDetails/locationOfGoods/LocationOfGoodsPostalCodePage.scala

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package pages.routeDetails.locationOfGoods
+
+import controllers.routeDetails.locationOfGoods.routes
+import models.{Address, Mode, PostalCodeAddress, UserAnswers}
+import pages.QuestionPage
+import pages.sections.routeDetails.LocationOfGoodsSection
+import play.api.libs.json.JsPath
+import play.api.mvc.Call
+
+case object LocationOfGoodsPostalCodePage extends QuestionPage[PostalCodeAddress] {
+
+  override def path: JsPath = LocationOfGoodsSection.path \ toString
+
+  override def toString: String = "locationOfGoodsPostalCode"
```

Comment:
Consider changing to `postalCode` to make interpreting the user answers json easier. We know it's nested inside `locationOfGoods`